train/tr11-200B-ml/tr11-200B-ml.slurm

```diff
@@ -53,34 +53,41 @@ export TRANSFORMERS_OFFLINE=1
 MASTER_ADDR=$(scontrol show hostnames $SLURM_JOB_NODELIST | head -n 1)
 MASTER_PORT=6000
 
+# Expected Throughput: 130.21 TFLOPS, Speed: 124.75 Sec/it
 GPUS_PER_NODE=8
-NNODES=$SLURM_NNODES
+NNODES=$SLURM_NNODES  # Should be 48
 
-PP_SIZE=10
+PP_SIZE=6
 TP_SIZE=8
 
-MICRO_BATCH_SIZE=1
-GLOBAL_BATCH_SIZE=2048
-TRAIN_SAMPLES=195_312_500 # 400B tokens
+MICRO_BATCH_SIZE=4
+GLOBAL_BATCH_SIZE=1568  # 3.21M tokens. TODO: check this number doesn't break constraints
```

Comment:
why not 2048? we haven't been benchmarking 3/4 of 2048 - potentially will make the pipeline less efficient -> slower throughput.

discuss on slack?

Reply:
ok. It is probably fine to use GBS=2048 but we will need to increase `--rampup-batch-size` a bit. I will post on slack tomorrow morning.

Reply:
we can also benchmark this new suggestion and then decide. I'm out of nodes right now, so can do tomorrow.